protocols/imap/src/test/java/org/apache/james/imap/processor/base/MailboxEventAnalyserTest.java

```diff
@@ -208,7 +208,9 @@ public void testShouldSetUidWhenSystemFlagChange() {
 
         testee.event(update);
 
-       assertThat(testee.flagUpdateUids().iterator()).containsExactly(UID);
+       assertThat(testee.flagUpdateUids().iterator())
```

Comment:
you can remove `.iterator()` then able to remove `.toIterable()`. Similar to some tests below

Reply:
Let me check :)